subprojects/plugin-use/src/integTest/groovy/org/gradle/plugin/repository/ResolvingFromMultipleCustomPluginRepositorySpec.groovy

```diff
@@ -75,13 +74,15 @@ class ResolvingFromMultipleCustomPluginRepositorySpec extends AbstractDependency
         settingsFile << """
             pluginManagement {
                 repositories {
-                    ${repositories.collect {
-                        if (it instanceof MavenFileRepository) {
-                            "maven { url '${it.uri}' }"
-                        } else {
-                            "ivy { url '${it.uri}' }"
-                        }
-                      }.join('\n')}
+                    ${
```

Comment:
I think I have it formatting the entire file instead of just lines I changed.  I will look into it.